stories/primer/tooltip_component_stories.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Primer::TooltipComponentStories < ViewComponent::Storybook::Stories
+  layout "storybook_preview"
```

Comment:
This is currently crashing locally, we think because of https://github.com/primer/view_components/issues/90. I'm looking into it.